Index: src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/App.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/App.java	(revision 67)
+++ src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/App.java	(revision )
@@ -11,19 +11,14 @@
         System.out.println(Date);
 
         System.out.println("Number of days: " + Date.getMonth().getDays(Date.getMonth().getMonthNumber(), Date.getYear().isLeapYear()));
-        //получение количества дней в месяце, с учетом високосности года
 
         System.out.println("Day of week: " + Date.getDayOfWeek());
-        //получение дня недели по заданной дате
 
         System.out.println("Day index: " + Date.getDayOfYear());
-        //получение дня недели по заданной дате
 
-        java.util.Date date = new java.util.Date(2016, 1, 1); //дата для сравнения
+        java.util.Date date = new java.util.Date(2016, 10, 10);
         System.out.println("Days between: " + Date.daysBetween(date));
-        //получеие количества дней в заданном временном промежутке.
 
         System.out.println("Value of index: " + ua.org.oa.rumyancevv.homeTask1.dateNum.Date.DayOfWeek.valueOf(Date.getDayOfYear()));
-        //преобразование индекса дня недели в соответствующий элемент перечисления
     }
 }
\ No newline at end of file
Index: src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/Date.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/Date.java	(revision 67)
+++ src/main/java/ua/org/oa/rumyancevv/homeTask1/dateNum/Date.java	(revision )
@@ -141,7 +141,6 @@
 
         public int getDays(int monthNumber, boolean leapYear) {
             return java.time.Month.of(monthNumber).length(leapYear);
-            //Колличество дней в определенном месяце
         }
 
         @Override
@@ -152,14 +151,11 @@
 
     public class Year {
 
-        //год
-
         private int year;
         private boolean leapYear;
 
         public Year(int year) {
             setYear(year);
-            //проверка года на високосность
             leapYear = java.time.Year.isLeap(year);
         }
 
@@ -197,7 +193,6 @@
     }
 
     public DayOfWeek getDayOfWeek() {
-        //День недели по заданной дате
         Calendar c = Calendar.getInstance();
         c.set(Year.getYear(), Month.getMonthNumber() - 1, Day.getDayNumber() - 1);
         int dayOfWeek = c.get(Calendar.DAY_OF_WEEK);
@@ -206,7 +201,6 @@
 
 
     public int getDayOfYear() {
-        //Неделя года по заданной дате
         Calendar c = Calendar.getInstance();
         c.set(Year.getYear(), Month.getMonthNumber() - 1, Day.getDayNumber() - 1);
         return c.get(Calendar.DAY_OF_WEEK) - 1;
@@ -214,7 +208,6 @@
     }
 
     public int daysBetween(java.util.Date date) {
-        //Колличество дней в заданном временном промежутке
 
         Calendar c = Calendar.getInstance();
         c.set(Year.getYear(), Month.getMonthNumber() - 1, Day.getDayNumber());
